certification/internal/policy/operator/operator.go

```diff
@@ -0,0 +1,64 @@
+package operator
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+func getAnnotationsFromBundle(mountedDir string) (map[string]string, error) {
+	log.Trace("reading annotations file from the bundle")
+	log.Debug("mounted directory is ", mountedDir)
+	annotationsFilePath := path.Join(mountedDir, "metadata", "annotations.yaml")
+
+	fileContents, err := os.ReadFile(annotationsFilePath)
+	if err != nil {
+		log.Error("fail to read metadata/annotation.yaml file in bundle")
+		return nil, err
+	}
+
+	annotations, err := extractAnnotationsBytes(fileContents)
+	if err != nil {
+		log.Error("metadata/annotations.yaml found but is malformed")
+		return nil, err
+	}
+
+	return annotations, nil
+}
+
+// extractAnnotationsBytes reads the annotation data read from a file and returns the expected format for that yaml
+// represented as a map[string]string.
+func extractAnnotationsBytes(annotationBytes []byte) (map[string]string, error) {
+	type metadata struct {
+		Annotations map[string]string
+	}
+
+	if len(annotationBytes) == 0 {
+		return nil, errors.ErrEmptyAnnotationFile
+	}
+
+	var bundleMeta metadata
+	if err := yaml.Unmarshal(annotationBytes, &bundleMeta); err != nil {
+		log.Error("metadata/annotations.yaml found but is malformed")
+		return nil, err
+	}
+
+	return bundleMeta.Annotations, nil
+}
+
+// Annotation() accepts the annotations map and searches for the specified annotation corresponding
```

Comment:
Nit: this doc reference should be `annotation`from `Annotation()`

Reply:
Will do.